front/src/modules/ui/board/components/__tests__/Board.test.ts

```diff
@@ -1,59 +1,53 @@
-import { DropResult } from '@hello-pangea/dnd';
-
-import { getOptimisticlyUpdatedBoard } from '../Board';
-
+// TODO: refactor this test with Recoil
 describe('getOptimisticlyUpdatedBoard', () => {
```

Comment:
This method is replaced by the logic in `front/src/modules/ui/board/hooks/useUpdateBoardCardIds.ts`